internal/data/receivers_test.go

```diff
@@ -448,7 +448,7 @@ func Test_ReceiversModel_GetAll(t *testing.T) {
 			require.Error(t, err, "not in transaction")
 		}()
 
-		receivers, err := receiverModel.GetAll(ctx, dbTx, &QueryParams{})
+		receivers, err := receiverModel.GetAll(ctx, dbTx, &QueryParams{}, QueryTypeSelectPaginated)
```

Comment:
Should the query type be `QueryTypeSelectAll`? I don't think it matters for the correctness of this test since there are no receivers.

Reply:
Same applies to the other tests under `Test_ReceiversModel_GetAll`.

Reply:
The default behaviour for `GetAll()` before I add that parameter was `QueryTypeSelectPaginated`. That's why I kept it the same. The method GetAll was doing paginated fetches. Adding `QueryTypeSelectAll` introduced non-paginated fetches.